django/econsensus/publicweb/templates/decision_list.html

```diff
@@ -66,8 +66,9 @@
                             {% if tab != 'actionitems' %}
                                 <td class="id"><a href="{% url 'publicweb_item_detail' object.id %}"><span class="iconified">{{ object.id }}</span></a></td>
                                 <td>
-                                    <a href="{% if request.user|is_watching:object %}{% url 'remove_watcher' object.id %}{% else %}{% url 'add_watcher' object.id %}{% endif %}?next={{ request.path_info }}">
-                                        <input type="checkbox"{% if request.user|is_watching:object %} checked="True"{% endif %} />
+                                    <a class='toggle watch'
+                                       href="{% if request.user|is_watching:object %}{% url 'remove_watcher' object.id %}{% else %}{% url 'add_watcher' object.id %}{% endif %}?next={{ request.path_info }}">
+                                        {% if request.user|is_watching:object %}&#x2611;{% else %}&#x2610;{% endif %}
```

Comment:
After some research it turns out that the W3C discourages using title attribute for links. See the warning under: http://www.w3.org/TR/html/dom.html#the-title-attribute

The W3C suggested approach is to hide part of the link text by moving it out of the viewport in a 1px box: http://www.w3.org/TR/2012/NOTE-WCAG20-TECHS-20120103/C7

I realise this is from 2012, though I can't find anything that says otherwise. I'm not particularly fond of this technique - still, if it is the recommended one I guess we should go ahead. What do you think?

Either way, I guess the text should be more descriptive. Something like 'watch/unwatch <item excerpt>'.